src/seedu/addressbook/data/person/Name.java

```diff
@@ -36,6 +36,23 @@ public static boolean isValidName(String test) {
         return test.matches(NAME_VALIDATION_REGEX);
     }
 
+
+    /**
```

Comment:
Need to have something useful in header comments. 
Empty comments like this are better not maintained.